docs/source/write_first_app.rst

```diff
@@ -1,4 +1,4 @@
-Writing Your First Application
+Running A Fabric Application
```

Comment:
I am more sympathetic to this title change. Running a Fabric application is more descriptive than writing your first app. But I would check and update the links to the article as well

Reply:
Agreed.